apps/ledger-live-desktop/src/config/urls.ts

```diff
@@ -153,6 +154,8 @@ export const urls = {
   },
   solana: {
     staking: "https://support.ledger.com/article/4731749170461-zd",
+    splTokenInfo:
+      "https://support.ledger.com/article/Verify-Solana-Address-from-Token-Account-Address",
```

Comment:
I can't see the article when going to this url. Is it expected ?
![Screenshot 2025-01-21 at 10 38 45](https://github.com/user-attachments/assets/fe05b1d1-42df-49d7-adfd-16734a653b25)

Reply:
It's expected right now and I asked to know when it's going to be up

Reply:
Alright thanks !